portfolio/src/main/webapp/index.html

```diff
@@ -18,6 +18,8 @@
       <img src="images/profile.jpeg" alt="profile" class="center">
       <h2><div id="changeText"></div></h2>
       <p>Fill in later!</p>
+      <button onclick="getHello()">Click me!</button>
```

Comment:
Generally we ban inline event handlers since it ties our code directly into the HTML structure, among other reasons. You can attach an event handler to a button in JavaScript with addEventListener e.g.

myButtonElement.addEventListener('click', (e) => {
  getHello();
});